net-tools/netwatch/src/netmon.rs

```diff
@@ -30,27 +29,27 @@ use self::actor::{Actor, ActorMessage};
 #[derive(Debug)]
 pub struct Monitor {
     /// Task handle for the monitor task.
-    handle: JoinHandle<()>,
+    #[allow(dead_code)]
+    handle: AbortOnDropHandle<()>,
```

Comment:
```suggestion
    _handle: AbortOnDropHandle<()>,
```
?

Reply:
it's actually used, which is why I don't want to use `_handle`

Reply:
then it doesn't need `#[allow(dead_code)]`?

Reply:
but rust is too stupid..it gets constructed and it has drop semantics.. but the compiler doesn't understand that as usage

Reply:
Or are we having a philosophical disagreement on what unused means?

My interpretation is the opposite in that case, to me it is definitely not dead code.  `_thing` does not mean something is unused, `let _guard = thing();` is an entirely common pattern, this is just the same but in a struct.

Reply:
you are right, will change

Reply:
fixed